### PR TITLE
workbench bug fix

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR
Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
add i18n for workbench
remove http protocol header for logo url

#### Specified Reivewers:
/assgin @sfwn @kakj-go

### CODE REVIEW

**Review by kakj-go — APPROVED** [review]

File: modules/openapi/api/apis/openapi/component_protocol/render.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/erda-project/erda/pkg/httpclient"
 	"github.com/erda-project/erda/pkg/httpserver"
 	"github.com/erda-project/erda/pkg/httpserver/errorresp"
+	i18nPkg "github.com/erda-project/erda/pkg/i18n"
```
Comment (sfwn) [inline modules/openapi/api/apis/openapi/component_protocol/render.go:35]: Use all lowercase in pkg name.

Comment (sfwn) [reply modules/openapi/api/apis/openapi/component_protocol/render.go:35]: You should merge "github.com/erda-project/erda/modules/openapi/i18n".

File: modules/openapi/component-protocol/scenarios/home-page-content/components/emptyOrgContent/render.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/erda-project/erda/apistructs"
 	protocol "github.com/erda-project/erda/modules/openapi/component-protocol"
+	i18n2 "github.com/erda-project/erda/modules/openapi/component-protocol/scenarios/home-page-content/i18n"
```
Comment (sfwn) [inline modules/openapi/component-protocol/scenarios/home-page-content/components/emptyOrgContent/render.go:25]: ugly pkg name.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/openapi/api/apis/openapi/component_protocol/render.go (hunk 1 above)
Comment (chengjoey) [reply modules/openapi/api/apis/openapi/component_protocol/render.go]: i think it should merge "github.com/erda-project/erda/modules/openapi/i18n" later,or build another task or bug ...

**Review by sfwn — APPROVED** [review]

**Pull request merged**